pkgs/development/compilers/bluespec-bsc/default.nix

```diff
@@ -0,0 +1,104 @@
+{ stdenv
+, fetchFromGitHub
+, fetchpatch
+, autoconf
+, automake
+, bison
+, flex
+, fontconfig
+, ghc
+, glibc
+, gmp
+, gperf
+, haskell
+, libX11
+, libpoly
+, perl
+, pkgconfig
+, verilog
+, xorg
+, zlib
+}:
+
+let
+  gmp-static = gmp.override { withStatic = true; };
+  ghcWithPackages = haskell.packages.ghc844.ghc.withPackages (g: (with g; [old-time regex-compat syb]));
+in stdenv.mkDerivation rec {
+  pname = "bsc";
```

Comment:
`s/bsc/bluespec/`, imo

Reply:
made it consistent with the folder it's in.